src/main/java/com/michelin/ns4kafka/service/executor/TopicAsyncExecutor.java

```diff
@@ -194,10 +197,6 @@ public void deleteTopic(Topic topic) throws InterruptedException, ExecutionExcep
 
         log.info("Success deleting topic {} on {}", topic.getMetadata().getName(),
             managedClusterProperties.getName());
-
-        if (managedClusterProperties.isConfluentCloud() && !topic.getSpec().getTags().isEmpty()) {
```

Comment:
This has been modified by https://github.com/michelin/ns4kafka/pull/400, right? It should not appear as a code change. Can you try to rebase your branch on master?